content/logs/index.md

```diff
@@ -147,22 +194,22 @@ However, if your logs defines one of the following attributes Datadog considers
 * `eventTime`
 * `published_date`
```

Comment:
What if more than one of these is present? Which takes precedence?

Reply:
asking

Reply:
Sequential order is applied, the first that matches is considered as the log's official date.